web/cypress/e2e/guest.spec.ts

```diff
@@ -31,36 +28,9 @@ describe("Guest Dashboard [feature] [all] [group2]", () => {
     onDashboardPage.getEditProfileLink().should("exist");
 
     cy.get('[data-testid="needs-account-alert"]').should("be.visible");
-    cy.wait(7000);
-    cy.get('[data-testid="needs-account-alert"]').should("not.exist");
 
     completeBusinessStructureTask({ legalStructureId });
 
-    // answer deferred question to get local-requirements task
```

Comment:
Same question as above.

Reply:
This industry changed from `home-contractor` to `cosmetology`. The home-based business no longer appears. Guest mode also needs a revisit as part of the Cypress work.

Reply:
Added a comment to explore Guest Mode in Figma (the page we are explore what is in scope for Cypress) https://www.figma.com/design/HHxMhhiQRN0FkzbJKiTWAu/BFS-User-Flows?node-id=1232-26&t=QsToiRqpNXc7FQr9-1